pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -108,11 +107,6 @@ func TestStatusManager_set(t *testing.T) {
 	mapper := &fakeRESTMapper{}
 	status := New(client, mapper, "testing", "1.2.3")
 
-	co, err := getCO(client, "testing")
-	if !errors.IsNotFound(err) {
-		t.Fatalf("unexpected error (expected Not Found): %v", err)
-	}
```

Comment:
Only the first return value is unused. You should change `co` to `_` but leave everything else. Likewise below.